Dockerfile

```diff
@@ -1,119 +1,37 @@
-FROM manageiq/ruby
-MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
-
-## For SUI
-ARG REF=master
+FROM centos/httpd:latest
 
-## Set ENV, LANG only needed if building with docker-1.8
-ENV container=docker \
-    CONTAINER=true \
-    APACHE_CONF_DIR=/etc/httpd/conf.d \
-    APP_ROOT=/var/www/miq/vmdb \
-    PERSISTENT=/persistent \
-    APPLIANCE_ROOT=/opt/manageiq/manageiq-appliance \
-    SUI_ROOT=/opt/manageiq/manageiq-ui-service \
-    CONTAINER_SCRIPTS_ROOT=/opt/manageiq/container-scripts \
-    TERM=xterm
+MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
 
 ## Atomic/OpenShift Labels
 LABEL name="manageiq-apache" \
       vendor="ManageIQ" \
       version="2.4.6-45" \
       url="http://manageiq.org/" \
-      summary="ManageIQ appliance apache image" \
+      summary="ManageIQ httpd image" \
       description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
       io.k8s.display-name="ManageIQ Apache" \
       io.k8s.description="ManageIQ Apache is the front-end for the ManageIQ Appliance." \
       io.openshift.expose-services="443:https" \
       io.openshift.tags="ManageIQ-Apache,apache"
 
-## To cleanly shutdown systemd, use SIGRTMIN+3
-STOPSIGNAL SIGRTMIN+3
-
 ## Install EPEL repo, yum necessary packages for the build without docs, clean all caches
 RUN yum -y install centos-release-scl-rh && \
     yum -y install --setopt=tsflags=nodocs \
-                   git                     \
-                   net-tools               \
-                   nodejs                  \
                    httpd                   \
```

Comment:
If we're basing on `centos/httpd` rather than a base centos image we don't need to install the httpd rpm here.